src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -36,6 +36,7 @@ namespace Microsoft.Terminal.TerminalControl
 
         void ScrollViewport(Int32 viewTop);
         Int32 GetScrollOffset();
+        Int32 GetTermHeight();

```

Comment:
I'd maybe change the name to `GetViewHeight`, since the `Term` seems a little redundant

Reply:
I'm on it